test/assertions_test.rb

```diff
@@ -124,7 +124,7 @@ def test_assert_statsd_call
       end
     end
 
-    assert_assertion_triggered do
+    assert_no_assertion_triggered do
```

Comment:
one argument against it is that you can now no longer assert on the absence of tags.. Like maybe there is a test that says "if a, then this tag is present, and if not a, then this tag is not present"...